include/clang-c/CXCppInterOp.h

```diff
@@ -12,7 +12,7 @@
 #include <stddef.h>
 #include <stdint.h>
 
-LLVM_CLANG_C_EXTERN_C_BEGIN
+extern "C" {
```

Comment:
Can we include `clang-c/ExternC.h` which defines the macro instead?

Reply:
clang-c/ExternC.h is already included, it looks like this PR does not make sense.